app/services/container_dashboard_service.rb

```diff
@@ -22,60 +24,98 @@ def status
       :nodes      => {
         :count        => @ems.present? ? @ems.container_nodes.count : ContainerNode.count,
         :errorCount   => 0,
-        :warningCount => 0
+        :warningCount => 0,
+        :href         => get_url_to_entity(:container_node)
       },
       :containers => {
         :count        => @ems.present? ? @ems.containers.count : Container.count,
         :errorCount   => 0,
-        :warningCount => 0
+        :warningCount => 0,
+        :href         => get_url_to_entity(:container)
       },
       :registries => {
         :count        => @ems.present? ? @ems.container_image_registries.count : ContainerImageRegistry.count,
         :errorCount   => 0,
-        :warningCount => 0
+        :warningCount => 0,
+        :href         => get_url_to_entity(:container_image_registry)
       },
       :projects   => {
         :count        => @ems.present? ? @ems.container_projects.count : ContainerProject.count,
         :errorCount   => 0,
-        :warningCount => 0
+        :warningCount => 0,
+        :href         => get_url_to_entity(:container_project)
       },
       :pods       => {
         :count        => @ems.present? ? @ems.container_groups.count : ContainerGroup.count,
         :errorCount   => 0,
         :warningCount => 0,
+        :href         => get_url_to_entity(:container_group)
       },
       :services   => {
         :count        => @ems.present? ? @ems.container_services.count : ContainerService.count,
         :errorCount   => 0,
-        :warningCount => 0
+        :warningCount => 0,
+        :href         => get_url_to_entity(:container_service)
       },
       :images     => {
         :count        => @ems.present? ? @ems.container_images.count : ContainerImage.count,
         :errorCount   => 0,
-        :warningCount => 0
+        :warningCount => 0,
+        :href         => get_url_to_entity(:container_image)
       },
       :routes     => {
         :count        => routes_count,
         :errorCount   => 0,
-        :warningCount => 0
+        :warningCount => 0,
+        :href         => get_url_to_entity(:container_route)
       }
     }
   end
 
   def providers
-    [
-      {
-        :iconClass    => "pficon pficon-openshift",
-        :count        => ManageIQ::Providers::Openshift::ContainerManager.count,
-        :id           => "openshift",
-        :providerType => "OpenShift"
-      },
-      {
-        :iconClass    => "pficon pficon-kubernetes",
-        :count        => ManageIQ::Providers::Kubernetes::ContainerManager.count,
-        :id           => "kubernetes",
-        :providerType => "Kubernetes"
-      }
-    ]
+    if @ems.present?
+      if @ems.kind_of?(ManageIQ::Providers::Openshift::ContainerManager)
+        {
+          :iconClass    => "pficon pficon-openshift",
+          :id           => "openshift",
+          :providerType => "OpenShift"
```

Comment:
> @abonas feels to me that it would just make the code more difficult to understand.
> what did you have in mind?

I think it will be more clear and more maintainable iwith less copy paste.
What I meant is, for example:
providerType = "OpenShift"
id = providerType.downcase